tools/data_scripts/checksum_fn.sh

```diff
@@ -6,8 +6,11 @@ calculate_sha1sum() {
     SAVE_TO_PATH=$2
     echo $'\nCalculating checksum for' ${CHECKSUM_FOLDER}
     echo "Saving to ${SAVE_TO_PATH}"
+    # echo "Uncomment to see what the checksum is being computer for "
+    # echo $(find $CHECKSUM_FOLDER -type f ! -name '*checksum*' ! -name '*MD*' \
+    #    -not -path '*/\.*' -exec realpath --relative-to $CHECKSUM_FOLDER {} \; | sort -d)
```

Comment:
remove commented out lines?

Reply:
they are for specifically here for debugging - see line 9